app/controllers/doorkeeper/authorizations_with_role_check_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Doorkeeper
+  class AuthorizationsWithRoleCheckController < Doorkeeper::AuthorizationsController
+    def new
+      # current_resource_owner == current_user
```

Comment:
Can we delete these two lines? It looks like they may be tests that were commented out.